### PR TITLE
Refactor code

### PR DESCRIPTION
**Pull request by paramt**
### Changes
1. Moved all scripts inside `src`
1. Moved all text files into `data`
1. Added pytests inside `tests`
1. Added TravisCI integration

### TODO
Todo on the server before merging
- Change `app.py` location in `crontab`
- Use `python 3.7` from `crontab`
- Add `data` to `.git/info/exclude`

After merging
- Run tests to add the latest submission to `replied.txt` without falsely replying 
- Copy `../subscribed.txt` to `data/subscribed.txt`

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+language: python
+
```
Comment (RDIL) [inline .travis.yml]:
```suggestion
python: 3.7
```

**Review by RDIL — CHANGES_REQUESTED** [review]

**Review by RDIL — APPROVED** [review]

**Comment by RDIL** [comment]
Fixed the build

**Comment by paramt** [comment]
Any idea on what the [complex method issue](https://www.codefactor.io/repository/github/paramt/memeadviser/pull/9) found by CodeFactor means? Or do you think we should just ignore it for now?

**Comment by RDIL** [comment]
It thinks that the method is too long and should be split in to multiple utility functions

example:
```python
# right now
def dostuff():
    hihighdsifgh
    gsdghdfgdjhsg
    gfdosgig
    hihighdsifgh
    gsdghdfgdjhsg
    gfdosgig
    hihighdsifgh
    gsdghdfgdjhsg
    gfdosgig

# solution:
def base():
   hihigh()
   gsd()
   whateverelse()

def hihigh():
    hihighdsifgh

def gsd():    
gsdghdfgdjhsg
   
def whateverelse():
    gfdosgig

**Comment by paramt** [comment]
Ah, I get it... will fix after merging

**Pull request merged**